src/JasperBus.Marten.Tests/MartenSubscriptionRepositoryTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Linq;
+using Baseline;
+using JasperBus.Configuration;
+using JasperBus.Marten.Tests.Setup;
+using JasperBus.Runtime.Subscriptions;
+using JasperBus.Tests.Runtime.Subscriptions;
+using Marten;
+using Xunit;
+using Shouldly;
+
+namespace JasperBus.Marten.Tests
+{
+    public class MartenSubscriptionRepositoryTests : IDisposable
```

Comment:
I'd probably push you to use the IntegrationContext here and make this be more of an integration test. Might be a little less code and you'd be better able to handle design evolution